{"quota_remaining": 9385, "quota_max": 10000, "has_more": true, "items": [{"view_count": 22, "last_edit_date": 1477942600, "score": 0, "tags": ["python", "django"], "last_activity_date": 1477942600, "is_answered": false, "question_id": 40338045, "creation_date": 1477898821, "answer_count": 0, "link": "http://stackoverflow.com/questions/40338045/adding-a-second-url-pattern-in-django-causes-attribute-error", "title": "Adding a Second URL Pattern in Django Causes Attribute Error", "owner": {"link": "http://stackoverflow.com/users/1695366/james-biederbeck", "reputation": 18, "profile_image": "https://www.gravatar.com/avatar/abf1ed6400e6805aca658ce57fe6167a?s=128&d=identicon&r=PG", "display_name": "James Biederbeck", "user_id": 1695366, "user_type": "registered"}}, {"view_count": 33, "last_edit_date": 1477942564, "score": 0, "tags": ["python", "google-app-engine", "google-cloud-datastore"], "last_activity_date": 1477942564, "is_answered": false, "question_id": 22781726, "creation_date": 1396344449, "answer_count": 0, "link": "http://stackoverflow.com/questions/22781726/gql-query-returning-unexpected-empty-with-indexed-entries", "title": "Gql query returning unexpected empty with indexed entries", "owner": {"link": "http://stackoverflow.com/users/3062352/user3062352", "reputation": 1, "profile_image": "https://www.gravatar.com/avatar/d80fca7113c23f6b7c8e4b61e64c5d6b?s=128&d=identicon&r=PG&f=1", "display_name": "user3062352", "user_id": 3062352, "user_type": "registered"}}, {"view_count": 12, "last_edit_date": 1477942549, "score": 0, "tags": ["python", "pandas", "plot", "dataframe", "boxplot"], "last_activity_date": 1477942549, "is_answered": false, "question_id": 40349226, "creation_date": 1477942143, "answer_count": 0, "link": "http://stackoverflow.com/questions/40349226/how-to-plot-on-box-plot-2-data-from-different-data-frama-python", "title": "How to plot on box plot 2 data from different data frama - Python", "owner": {"link": "http://stackoverflow.com/users/6781190/rafael-rodrigues-santos", "reputation": 20, "profile_image": "https://lh6.googleusercontent.com/-MTsq6jGgD3g/AAAAAAAAAAI/AAAAAAAAAYo/GuiQztfX8ew/photo.jpg?sz=128", "display_name": "Rafael Rodrigues Santos", "user_id": 6781190, "user_type": "registered"}}, {"view_count": 10, "link": "http://stackoverflow.com/questions/40349281/rabbitmq-pika-connection-errors-and-inconsistency", "owner": {"accept_rate": 70, "link": "http://stackoverflow.com/users/673600/techdisrupt", "reputation": 609, "profile_image": "https://www.gravatar.com/avatar/fa996ef5abf6c152d13ee49ba1074d44?s=128&d=identicon&r=PG", "display_name": "techdisrupt", "user_id": 673600, "user_type": "registered"}, "tags": ["python", "rabbitmq", "pika"], "last_activity_date": 1477942403, "is_answered": false, "score": 0, "question_id": 40349281, "creation_date": 1477942403, "answer_count": 0, "title": "RabbitMQ pika connection errors and inconsistency"}, {"view_count": 14, "link": "http://stackoverflow.com/questions/40349255/django-prepend-www-when-http-is-hardcoded-into-backlink", "owner": {"accept_rate": 83, "link": "http://stackoverflow.com/users/3200391/studio-rooster", "reputation": 251, "profile_image": "https://www.gravatar.com/avatar/dc8c798b0aaf54577e388f7bc177093c?s=128&d=identicon&r=PG", "display_name": "Studio Rooster", "user_id": 3200391, "user_type": "registered"}, "tags": ["python", "django"], "last_activity_date": 1477942256, "is_answered": false, "score": 0, "question_id": 40349255, "creation_date": 1477942256, "answer_count": 0, "title": "Django PREPEND_WWW when http:// is hardcoded into backlink"}, {"view_count": 14, "link": "http://stackoverflow.com/questions/40349242/how-to-deal-with-incorrect-json-format-simplejson-scanner-jsondecodeerror-ex", "owner": {"link": "http://stackoverflow.com/users/6616052/pythonista", "reputation": 8, "profile_image": "https://www.gravatar.com/avatar/d1f12aa3ae580eaa7cfb90a2400fc220?s=128&d=identicon&r=PG&f=1", "display_name": "pythonista", "user_id": 6616052, "user_type": "registered"}, "tags": ["python", "json", "python-2.7"], "last_activity_date": 1477942216, "is_answered": false, "score": 0, "question_id": 40349242, "creation_date": 1477942216, "answer_count": 0, "title": "How to deal with incorrect json format ? (simplejson.scanner.JSONDecodeError: Expecting value: line 1 column 1 (char 0))"}, {"view_count": 23, "last_edit_date": 1477942171, "score": 0, "tags": ["python", "caffe", "pycaffe"], "last_activity_date": 1477942171, "is_answered": false, "question_id": 40278047, "creation_date": 1477551198, "answer_count": 0, "link": "http://stackoverflow.com/questions/40278047/dnn-pruning-and-fine-tuningretraining-implement-on-pycaffe", "title": "DNN pruning and fine-tuning(retraining) implement on pycaffe", "owner": {"link": "http://stackoverflow.com/users/7078466/sandman", "reputation": 8, "profile_image": "https://www.gravatar.com/avatar/8c5d3018a660b84d8b7d398e47204c90?s=128&d=identicon&r=PG&f=1", "display_name": "sandman", "user_id": 7078466, "user_type": "registered"}}, {"view_count": 10, "last_edit_date": 1477942015, "score": -1, "tags": ["python", "ubuntu", "heroku", "flask"], "last_activity_date": 1477942015, "is_answered": false, "question_id": 40348649, "creation_date": 1477939831, "answer_count": 0, "link": "http://stackoverflow.com/questions/40348649/flask-app-template-changes-do-not-apply-even-after-server-restarts", "title": "Flask App template Changes do not apply, even after server restarts", "owner": {"link": "http://stackoverflow.com/users/7028341/gdmurray", "reputation": 1, "profile_image": "https://www.gravatar.com/avatar/70d66f4b45332f158642e686405aec88?s=128&d=identicon&r=PG&f=1", "display_name": "gdmurray", "user_id": 7028341, "user_type": "registered"}}, {"view_count": 8, "last_edit_date": 1477941828, "score": 0, "tags": ["python", "selenium", "selenium-chromedriver"], "last_activity_date": 1477941828, "is_answered": false, "question_id": 40348339, "creation_date": 1477938687, "answer_count": 0, "link": "http://stackoverflow.com/questions/40348339/when-initially-creating-a-chromedriver-in-python-http-client-badstatusline", "title": "When initially creating a chromedriver in python, http.client.BadStatusLine: &#39;&#39; is thrown", "owner": {"link": "http://stackoverflow.com/users/5233367/brydenr", "reputation": 162, "profile_image": "https://www.gravatar.com/avatar/60dc3184f9d77e7d07bfa556739d55fc?s=128&d=identicon&r=PG&f=1", "display_name": "Brydenr", "user_id": 5233367, "user_type": "registered"}}, {"view_count": 12, "link": "http://stackoverflow.com/questions/40349110/unable-to-install-networkit-using-sudo-pip3-install-networkit", "owner": {"link": "http://stackoverflow.com/users/7096758/ray", "reputation": 1, "profile_image": "https://www.gravatar.com/avatar/9999921dd9281860c47e651a316135dc?s=128&d=identicon&r=PG&f=1", "display_name": "ray", "user_id": 7096758, "user_type": "registered"}, "tags": ["python"], "last_activity_date": 1477941676, "is_answered": false, "score": 0, "question_id": 40349110, "creation_date": 1477941676, "answer_count": 0, "title": "Unable to install Networkit using sudo pip3 install networkit"}, {"view_count": 8, "last_edit_date": 1477941475, "score": 0, "tags": ["javascript", "python"], "last_activity_date": 1477941475, "is_answered": false, "question_id": 40348906, "creation_date": 1477940896, "answer_count": 0, "link": "http://stackoverflow.com/questions/40348906/python-dryscrape-scroll-down", "title": "Python-Dryscrape scroll down", "owner": {"accept_rate": 86, "link": "http://stackoverflow.com/users/6930377/dilli", "reputation": 115, "profile_image": "https://www.gravatar.com/avatar/72345ae8dcbde9beac98f975d909e84a?s=128&d=identicon&r=PG&f=1", "display_name": "Dilli", "user_id": 6930377, "user_type": "registered"}}, {"view_count": 11, "link": "http://stackoverflow.com/questions/40349037/connecting-to-ms-dynamics-ax-using-component-object-model-in-pyhton", "owner": {"link": "http://stackoverflow.com/users/6668291/d-wi%c5%9bniewski", "reputation": 3, "profile_image": "https://www.gravatar.com/avatar/adbaab54ae8f278afbff6b05a17fbbf8?s=128&d=identicon&r=PG&f=1", "display_name": "D. Wi\u015bniewski", "user_id": 6668291, "user_type": "registered"}, "tags": ["python", "com", "axapta", "microsoft-dynamics", "win32com"], "last_activity_date": 1477941426, "is_answered": false, "score": 0, "question_id": 40349037, "creation_date": 1477941426, "answer_count": 0, "title": "Connecting to MS Dynamics AX using Component Object Model in Pyhton"}, {"view_count": 6, "link": "http://stackoverflow.com/questions/40348656/editing-camera-settings-by-using-opencv", "owner": {"accept_rate": 25, "link": "http://stackoverflow.com/users/3337902/user3337902", "reputation": 21, "profile_image": "https://www.gravatar.com/avatar/f60c0ccc5b7a6654dc61a5e72a013d2f?s=128&d=identicon&r=PG&f=1", "display_name": "user3337902", "user_id": 3337902, "user_type": "registered"}, "tags": ["python", "opencv", "camera-calibration"], "last_activity_date": 1477939858, "is_answered": false, "score": 0, "question_id": 40348656, "creation_date": 1477939858, "answer_count": 0, "title": "Editing Camera Settings by using Opencv"}, {"view_count": 8, "last_edit_date": 1477939733, "score": -1, "tags": ["python", "mongodb", "facebook-graph-api"], "last_activity_date": 1477939733, "is_answered": false, "question_id": 40348103, "creation_date": 1477937813, "answer_count": 0, "link": "http://stackoverflow.com/questions/40348103/python-and-facebook-api-graph-keywords-filter", "title": "Python and Facebook API Graph - Keywords Filter", "owner": {"link": "http://stackoverflow.com/users/6926525/leandro-s-matos", "reputation": 7, "profile_image": "https://www.gravatar.com/avatar/583c9aaf83a85a8be95ff979e517a212?s=128&d=identicon&r=PG&f=1", "display_name": "Leandro S. Matos", "user_id": 6926525, "user_type": "registered"}}, {"view_count": 7, "link": "http://stackoverflow.com/questions/40348607/sklearn-ridge-indexerror-tuple-index-out-of-range", "owner": {"accept_rate": 89, "link": "http://stackoverflow.com/users/1658445/nsr", "reputation": 166, "profile_image": "https://www.gravatar.com/avatar/ea30ce393706e5167fe03b79ded94939?s=128&d=identicon&r=PG", "display_name": "NSR", "user_id": 1658445, "user_type": "registered"}, "tags": ["python", "linear-regression"], "last_activity_date": 1477939676, "is_answered": false, "score": 0, "question_id": 40348607, "creation_date": 1477939676, "answer_count": 0, "title": "sklearn Ridge IndexError: tuple index out of range"}, {"view_count": 7, "link": "http://stackoverflow.com/questions/40348582/how-to-set-timestamp-in-past-manually-with-rollbar", "owner": {"accept_rate": 87, "link": "http://stackoverflow.com/users/197546/daniel", "reputation": 6450, "profile_image": "https://www.gravatar.com/avatar/3f3f0b68c6bba7cc2cf07f51c1a1afcd?s=128&d=identicon&r=PG", "display_name": "Daniel", "user_id": 197546, "user_type": "registered"}, "tags": ["python", "rollbar"], "last_activity_date": 1477939596, "is_answered": false, "score": 0, "question_id": 40348582, "creation_date": 1477939596, "answer_count": 0, "title": "how to set timestamp in past (manually) with rollbar"}, {"view_count": 4, "link": "http://stackoverflow.com/questions/40348504/graphviz-not-rendering-decision-tree-in-jupyter-notebook", "owner": {"accept_rate": 71, "link": "http://stackoverflow.com/users/6534621/amber", "reputation": 51, "profile_image": "https://www.gravatar.com/avatar/a7eabdeaf80485aea3e973a1b742c54b?s=128&d=identicon&r=PG&f=1", "display_name": "Amber", "user_id": 6534621, "user_type": "registered"}, "tags": ["python", "python-2.7", "graphviz", "jupyter-notebook"], "last_activity_date": 1477939353, "is_answered": false, "score": 0, "question_id": 40348504, "creation_date": 1477939353, "answer_count": 0, "title": "Graphviz not rendering Decision Tree in Jupyter Notebook"}, {"view_count": 6, "last_edit_date": 1477939303, "score": 0, "tags": ["python", "osx", "sqlite3", "pip"], "last_activity_date": 1477939303, "is_answered": false, "question_id": 40347590, "creation_date": 1477935838, "answer_count": 0, "link": "http://stackoverflow.com/questions/40347590/error-loading-either-pysqlite2-or-sqlite3-modules-in-mac", "title": "error loading either pysqlite2 or sqlite3 modules in Mac", "owner": {"accept_rate": 44, "link": "http://stackoverflow.com/users/1081319/parameswar", "reputation": 205, "profile_image": "https://www.gravatar.com/avatar/641515d110f3b086a99c375c7e63911e?s=128&d=identicon&r=PG", "display_name": "parameswar", "user_id": 1081319, "user_type": "registered"}}, {"view_count": 11, "link": "http://stackoverflow.com/questions/40348468/how-to-know-a-specific-test-failed-in-python-selenium-testrail", "owner": {"link": "http://stackoverflow.com/users/7096579/yago", "reputation": 1, "profile_image": "https://www.gravatar.com/avatar/ad3163c3c982bf07e4b3b3fb3d264700?s=128&d=identicon&r=PG&f=1", "display_name": "Yago", "user_id": 7096579, "user_type": "registered"}, "tags": ["python", "selenium", "testrail"], "last_activity_date": 1477939200, "is_answered": false, "score": 0, "question_id": 40348468, "creation_date": 1477939200, "answer_count": 0, "title": "How to know a specific test failed in python selenium + testrail"}, {"view_count": 17, "last_edit_date": 1477939139, "score": 0, "tags": ["python", "if-statement"], "last_activity_date": 1477939139, "is_answered": false, "question_id": 40347998, "creation_date": 1477937394, "answer_count": 0, "link": "http://stackoverflow.com/questions/40347998/why-isnt-my-python-shell-indenting-automatically", "title": "Why isn&#39;t my python shell indenting automatically?", "owner": {"link": "http://stackoverflow.com/users/7080788/steve-smith", "reputation": 6, "profile_image": "https://www.gravatar.com/avatar/aed9900c1c78a69993a89374809906a5?s=128&d=identicon&r=PG&f=1", "display_name": "Steve Smith", "user_id": 7080788, "user_type": "registered"}}, {"view_count": 31, "last_edit_date": 1477938905, "score": 2, "tags": ["python", "multithreading"], "last_activity_date": 1477938905, "is_answered": false, "question_id": 40347789, "creation_date": 1477936615, "answer_count": 0, "link": "http://stackoverflow.com/questions/40347789/variables-being-set-to-none-in-python-multithreaded-app", "title": "Variables being set to None in Python multithreaded app", "owner": {"accept_rate": 94, "link": "http://stackoverflow.com/users/311901/jordan-feldstein", "reputation": 4854, "profile_image": "https://www.gravatar.com/avatar/ae735b7993ad5c05647c98c509a2a765?s=128&d=identicon&r=PG", "display_name": "Jordan Feldstein", "user_id": 311901, "user_type": "registered"}}, {"view_count": 10, "last_edit_date": 1477938880, "score": 0, "tags": ["python", "recursion", "copy", "deep-copy"], "last_activity_date": 1477938880, "is_answered": false, "question_id": 40348262, "creation_date": 1477938399, "answer_count": 0, "link": "http://stackoverflow.com/questions/40348262/deepcopy-object-with-cyclic-references", "title": "__deepcopy__ object with cyclic references", "owner": {"accept_rate": 80, "link": "http://stackoverflow.com/users/232371/ben-usman", "reputation": 1107, "profile_image": "https://www.gravatar.com/avatar/2087f266418f98e871dac8d18f3c150e?s=128&d=identicon&r=PG", "display_name": "Ben Usman", "user_id": 232371, "user_type": "registered"}}, {"view_count": 11, "link": "http://stackoverflow.com/questions/40348363/400-error-msg-when-using-requests-post-in-python-3", "owner": {"accept_rate": 38, "link": "http://stackoverflow.com/users/5399672/bwoods", "reputation": 53, "profile_image": "https://www.gravatar.com/avatar/4985cf34085fca89a593804173d6d9b7?s=128&d=identicon&r=PG&f=1", "display_name": "Bwoods", "user_id": 5399672, "user_type": "registered"}, "tags": ["python", "rest", "client"], "last_activity_date": 1477938791, "is_answered": false, "score": 1, "question_id": 40348363, "creation_date": 1477938791, "answer_count": 0, "title": "400 error msg when using requests.post in python 3"}, {"view_count": 22, "last_edit_date": 1477938209, "score": 0, "tags": ["python", "python-3.x", "case-folding"], "last_activity_date": 1477938209, "is_answered": false, "question_id": 40348174, "creation_date": 1477938098, "answer_count": 0, "link": "http://stackoverflow.com/questions/40348174/should-i-use-python-casefold", "title": "Should I use Python casefold?", "owner": {"link": "http://stackoverflow.com/users/5053822/flyinglightning", "reputation": 30, "profile_image": "https://lh6.googleusercontent.com/-aE3bwOS3uuo/AAAAAAAAAAI/AAAAAAAAAEo/BD-0Z5Az9M4/photo.jpg?sz=128", "display_name": "FlyingLightning", "user_id": 5053822, "user_type": "registered"}}, {"view_count": 10, "link": "http://stackoverflow.com/questions/40348185/comparing-svm-kernels-with-cross-validation", "owner": {"link": "http://stackoverflow.com/users/7096470/plv", "reputation": 1, "profile_image": "https://www.gravatar.com/avatar/8598399ec6a031d3b48ce770441d0374?s=128&d=identicon&r=PG&f=1", "display_name": "PLV", "user_id": 7096470, "user_type": "registered"}, "tags": ["python", "svm", "cross-validation"], "last_activity_date": 1477938135, "is_answered": false, "score": 0, "question_id": 40348185, "creation_date": 1477938135, "answer_count": 0, "title": "Comparing SVM-Kernels with cross - validation"}, {"view_count": 18, "last_edit_date": 1477938070, "score": 0, "tags": ["python", "pandas", "dataframe", "subset"], "last_activity_date": 1477938070, "is_answered": false, "question_id": 40348099, "creation_date": 1477937802, "answer_count": 0, "link": "http://stackoverflow.com/questions/40348099/how-do-i-freeze-a-pandas-dataframe-column-when-subsetting-the-data-frame", "title": "How do I freeze a pandas dataframe column when subsetting the data frame?", "owner": {"link": "http://stackoverflow.com/users/7032718/annieco2929", "reputation": 6, "profile_image": "https://www.gravatar.com/avatar/5a1b0ecb0e008a6599976f6b34795929?s=128&d=identicon&r=PG&f=1", "display_name": "annieco2929", "user_id": 7032718, "user_type": "registered"}}, {"view_count": 3, "link": "http://stackoverflow.com/questions/40348145/python-beautifulsoup-print-koi8-r-encoded-text-as-cyrillic", "owner": {"accept_rate": 60, "link": "http://stackoverflow.com/users/5425036/occams-chainsaw", "reputation": 47, "profile_image": "https://graph.facebook.com/907015839364711/picture?type=large", "display_name": "Occam&#39;s chainsaw", "user_id": 5425036, "user_type": "registered"}, "tags": ["python", "encoding", "beautifulsoup"], "last_activity_date": 1477937960, "is_answered": false, "score": 0, "question_id": 40348145, "creation_date": 1477937960, "answer_count": 0, "title": "Python BeautifulSoup print koi8-r encoded text as Cyrillic"}, {"view_count": 6, "link": "http://stackoverflow.com/questions/40348100/editing-django-admin-change-list-item", "owner": {"link": "http://stackoverflow.com/users/7069853/fri-gate", "reputation": 1, "profile_image": "https://www.gravatar.com/avatar/0412ab101370b938c033101d236d8efd?s=128&d=identicon&r=PG&f=1", "display_name": "Fri Gate", "user_id": 7069853, "user_type": "registered"}, "tags": ["python", "django", "python-2.7", "django-admin"], "last_activity_date": 1477937806, "is_answered": false, "score": 0, "question_id": 40348100, "creation_date": 1477937806, "answer_count": 0, "title": "Editing django admin change list item"}, {"view_count": 14, "link": "http://stackoverflow.com/questions/40348083/making-a-2d-histogram-from-a-2d-array", "owner": {"accept_rate": 80, "link": "http://stackoverflow.com/users/4381260/arland", "reputation": 17, "profile_image": "https://www.gravatar.com/avatar/079a3cfc59960cb7dc666ab9df873c8a?s=128&d=identicon&r=PG&f=1", "display_name": "Arland", "user_id": 4381260, "user_type": "registered"}, "tags": ["python", "arrays", "histogram2d"], "last_activity_date": 1477937746, "is_answered": false, "score": 0, "question_id": 40348083, "creation_date": 1477937746, "answer_count": 0, "title": "making a 2d histogram from a 2d array"}, {"view_count": 4, "link": "http://stackoverflow.com/questions/40348040/how-to-sign-xml-with-python-suds", "owner": {"link": "http://stackoverflow.com/users/3645619/user3645619", "reputation": 6, "profile_image": "https://graph.facebook.com/1045340438/picture?type=large", "display_name": "user3645619", "user_id": 3645619, "user_type": "registered"}, "tags": ["python", "soap", "suds"], "last_activity_date": 1477937598, "is_answered": false, "score": 1, "question_id": 40348040, "creation_date": 1477937598, "answer_count": 0, "title": "How to sign XML with python ( suds)"}]}